doc/userguide/upgrade.rst

```diff
@@ -34,6 +34,10 @@ also check all the new features that have been added but are not covered by
 this guide. Those features are either not enabled by default or require
 dedicated new configuration.
 
+Upgrading 7.0 to 8.0
+--------------------
+- SIP over TCP is detected
```

Comment:
"detected" seems to miss represent the features. SIP/TCP is detected, tracked, logged and available to rules through the sip keywords?

Reply:
doc updated.